activerecord/test/cases/base_test.rb

```diff
@@ -1412,6 +1412,14 @@ def test_default_values_are_deeply_dupped
     assert_not_includes SymbolIgnoredDeveloper.columns_hash.keys, "first_name"
   end
 
+  test "#columns_hash raises an error if the record has an empty table name" do
```

Comment:
This is a class method so I think `".columns_hash raises an error if the record has an empty table name"` is more appropriate.